src/components/ebay-education-notice/examples/with-footer.marko

```diff
@@ -0,0 +1,15 @@
+import type { Input as EducationNoticeInput } from "<ebay-education-notice>";
+export type Input = EducationNoticeInput;
+
+<ebay-education-notice ...input>
+    <@title>
+        <div>Educaiton title</div>
+    </@title>
+    <@footer>
```

Comment:
The CTA actually creates the `footer` since it's the primary (and currently only) element there.

Reply:
Not sure what this means.
Footer allows them to put what they want. We put a CTA there to show thats what it should have.

Reply:
Resolving for now, please reopen if needed and add more comments.

Reply:
Ok, that makes sense based on our approach. I was thinking the structural elements (i.e. `@footer`) would be put together automatically through internal conditionals based on the options passed in instead of the devs explicitly including them. Given our approach, what you have makes sense.